src/pull_request.ts

```diff
@@ -32,6 +32,10 @@ export async function handlePullRequest() {
 
   const octokit = initOctokit(config.githubToken);
 
+  if (shouldIgnorePullRequest(pull_request)) {
+    return;
+  }
```

Comment:
The function call `shouldIgnorePullRequest(pull_request)` assumes that `pull_request` is properly defined. Consider adding a null check to handle cases where the pull request object might be undefined:
```typescript
if (!pull_request) {
  error('Pull request object is undefined');
  return;
}

if (shouldIgnorePullRequest(pull_request)) {
  return;
}
```